### PR TITLE
Shorten CRD

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
Shorten CRD by remove description and unncessary propertyies.

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/pingcap/v1alpha1/types.go [hunk 1]
```diff
@@ -174,6 +174,7 @@ type TidbClusterSpec struct {
 
 	// Affinity of TiDB cluster Pods
 	// +optional
+	// +k8s:openapi-gen=false
```
Comment (DanielZhangQD) [inline pkg/apis/pingcap/v1alpha1/types.go:177]: Since no description now, we can revert the changes in *types.go.

Comment (Yisaer) [reply pkg/apis/pingcap/v1alpha1/types.go]: updated.
